<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AsientoListBase</class>
 <widget class="QWidget" name="AsientoListBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Listado de Asientos</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/BulmaCont32x32/images/pendientes/i_asiento.xpm</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages-recursos-graficos.qrc" >:/BulmaCont32x32/images/pendientes/i_asiento.xpm</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3Frame" name="m_botonera" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_imprimir" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Imprimir</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_filePrint.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_botonfiltrar" >
          <property name="toolTip" >
           <string>Filtrar</string>
          </property>
          <property name="whatsThis" >
           <string>Filtrar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_find.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_configurar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Configurar</string>
          </property>
          <property name="whatsThis" >
           <string>Configurar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_packageSettings.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Actualizar</string>
          </property>
          <property name="whatsThis" >
           <string>Actualizar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_reload.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>351</width>
            <height>39</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Ejercicio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mui_ejercicio" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel2" >
          <property name="text" >
           <string>Asientos cuyo nombre se parezca a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mui_nombreasiento" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabel2_2" >
            <property name="text" >
             <string>Asientos con saldo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mui_saldoasiento" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabel2_2_2" >
            <property name="text" >
             <string>Asientos con  apuntes con cantidad igual a</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mui_cantidadapunte" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="AsientoListSubform" name="mui_list" />
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends></extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>AsientoListSubform</class>
   <extends>QTableWidget</extends>
   <header>asientolistsubform.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages-recursos-graficos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_botonfiltrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>113</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel" >
     <x>155</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
